ledger-core/v2/conveyor/conveyor.go

```diff
@@ -182,7 +182,7 @@ func (p *PulseConveyor) mapToPulseSlotMachine(pn pulse.Number) (*PulseSlotMachin
 	switch {
 	case presentPN.IsUnknown():
 		// when no present pulse - all pulses go to future
-		return p.getFuturePulseSlotMachine(presentPN, futurePN), pn, Future, nil
```

Comment:
@cyraxred this one was broken, as far as I understand this is the fix

Reply:
nope, pls look into GetPresentPulse()

	if v == 0 {
		return pulse.Unknown, uninitializedFuture
	}

Reply:
this one fails later in getFuturePulseSlotMachine, because uninitializedFuture is not valid pulse

Reply:
will fix